libnixf/include/nixf/Basic/Range.h

```diff
@@ -28,6 +29,10 @@ class Position {
   friend bool operator<=(const Position &LHS, const Position &RHS) {
     return LHS < RHS || LHS == RHS;
   }
+
+  std::string to_string() const {
```

Comment:
Is this method intended solely for testing purposes? If so, I would prefer to test the position directly.

Reply:
Yes it is, i will remove it